app/scripts/metamask-controller.js

```diff
@@ -184,6 +184,7 @@ export default class MetamaskController extends EventEmitter {
     });
 
     this.assetsContractController = new AssetsContractController();
+    this.assetsContractController.configure({ provider: this.provider });
```

Comment:
Only a question so I can understand: why does this need to be set with a `configure` call, instead of just passing it with the constructor?

Reply:
you're right this can be done in the constructor 👍